### ISSUE TITLE
Not able to create a transaction_detection_rule

### ISSUE THREAD

**Issue by seyedk** [HarryEMartland/terraform-provider-appdynamics#31]
Hello Harry. With a lot of thanks for building such as nice provider, I have been struggling to configure the transaction detection rules. Can you provide a little more details on some more concrete examples? when I give the following inputs, I get "error creating transaction rule, no results returned". Here are the input parameters:

```

application_id = var.application_id

account_id = var.account_id

scope_id = "TEST"

name = "My test rule"

agent_type = "DOT_NET_APPLICATION_SERVER"

description = "My test health rule created with Terraform"

entry_point_type = "WCF"

http_uri_match_type = "MATCHES_REGEX"

http_uris = ["/.*"]

http_method = "GET"

```

**Comment by HarryEMartland** [comment]
You definatly have a scope called 'TEST' in that application? I think that was the next resource I wanted to add.

**Comment by HarryEMartland** [comment]
In fact refreshing myself of the blog post it looks like scope needs to be a UUID. I think you can get this from editing a scope in the AppD UI and looking at the network requests. There is a UUID as a query parameter for the `ScopeEditorDialog.html` request

**Comment by seyedk** [comment]
@HarryEMartland. Thanks a lot for the prompt response. As you mentioned, it requires the GUID of the scope. BTW, the query string parameter to the ScopeEditorDialog.html is not the right UUID. What I did was creating a new transaction detection rule from the UI and saved it. the request query string parameter had the correct scope ID (in GUID). Also, I noticed I was missing a character in the account Id (another GUID) as well! Thanks again for sharing :=)
